### DISCUSSION TITLE
ATTACK Navigator (Not Working) 2.3.10

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#2063] [Unsupported Versions] [answered]
The Attack Navigator wasn't properly populating when i did a Air Gap setup of 2.3 about a month ago. After updating to 2.3.10 the Navigator is still empty. I can provide screenshots if necessary. Any ideas on how to fix this?

**Comment by defensivedepth** [comment]
Do you see any errors when you restart nginx (`so-nginx-restart --force`)

At the very bottom left of the Nagivator app, you should see a version number - what version does it display?

Finally, is there a layer titled `Playbook`, but it's just empty?

**Reply by ColeVan** [reply]
No error when issuing the (so-nginx-restart --force) command. Here are my screenshots.
![image](https://user-images.githubusercontent.com/70167373/100105655-9c93ac00-2e35-11eb-8c98-075d7226d1d5.png)
![image](https://user-images.githubusercontent.com/70167373/100105741-b0d7a900-2e35-11eb-891f-c2e735e4596e.png)

**Comment by defensivedepth** [comment, marked as answer]
Thanks for the additional info. I believe this is because it has to pull down some initial files from the Internet, which obviously doesn't work for Airgap. I have created an issue to get this fixed for Airgap installs: https://github.com/Security-Onion-Solutions/securityonion/issues/2068
